test/hf/run_autohf.py

```diff
@@ -0,0 +1,507 @@
+    rm_home_result()
+
+
+def _exhaustive_sweep(console_args,
+                      jobid_config,
+                      autohf,
+                      wandb_utils,
+                      azure_utils=None,
+                      autohf_settings=None, ):
+    from flaml.nlp.hpo.hpo_searchspace import AutoHPOSearchSpace
+    console_args.space_mode = jobid_config.spa = "cus"
+    console_args.algo_mode = jobid_config.mod = "grid"
+    console_args.algo_name = jobid_config.alg = "grid"
+
+    gridunion_space = AutoHPOSearchSpace.from_model_and_dataset_name(
+        "uni",
+        jobid_config.pre,
+        jobid_config.presz,
+        jobid_config.dat,
+        jobid_config.subdat
+    )
+
+    gridunion_space["learning_rate"] = [float(x) for x in console_args.learning_rate]
+    gridunion_space["weight_decay"] = [float(x) for x in console_args.weight_decay]
+    _test_hpo(console_args, jobid_config, autohf, wandb_utils, azure_utils,
+              autohf_settings,
+              root_log_path=console_args.root_log_path,
+              **{"hpo_space": gridunion_space})
+
+def modelcard_exp(console_args):
+    if "hp1" in console_args.root_log_path:
+        console_args.seed_transformers = 42
+        evaluate_configs(autohf, console_args, [{"learning_rate": 3e-05, "per_device_train_batch_size": 16,
+                                             "num_train_epochs": 10, "warmup_ratio": 0.0,
+                                             "weight_decay": 0.1, "adam_epsilon": 1e-6}])
+    else:
+        console_args.seed_transformers = 41
+        evaluate_configs(autohf, console_args, [{"learning_rate": 1e-05, "per_device_train_batch_size": 32,
+                                             "num_train_epochs": 3, "warmup_ratio": 0.0,
+                                             "weight_decay": 0.0, "adam_epsilon": 1e-8}])
+
+
+if __name__ == "__main__":
+    from flaml.nlp.hpo.hpo_searchspace import AutoHPOSearchSpace
+    import itertools
+
+    console_args = load_dft_args()
+
+    jobid_config = JobID(console_args)
+    autohf = AutoTransformers()
+    wandb_utils = WandbUtils(is_wandb_on=False, wandb_key_path=console_args.key_path, jobid_config=jobid_config)
+    wandb_utils.set_wandb_per_run()
+
+    # _test_hpo(console_args, jobid_config, autohf, wandb_utils)
+
+    # search_base_and_search_lower_lr(console_args, jobid_config, autohf, wandb_utils)
+
+    # evaluate_small_best_configs_on_large(console_args, autohf)
+
+    # evaluate_large_best_configs_on_small(console_args, autohf)
+
+    # _exhaustive_sweep(console_args, jobid_config, autohf, wandb_utils)
+
+    # evaluate_configs(autohf, console_args)
+
+    if console_args.algo_mode.endswith("cv"):
+        evaluate_configs_cv(autohf, console_args, 10, wandb_utils)
+    elif console_args.algo_mode.endswith("hpo"):
+        _test_hpo(console_args, jobid_config, autohf, wandb_utils)
+    else:
+        modelcard_exp(console_args)
```

Comment:
What's the purpose of run_autohf.py and run_analysis.py respectively? Are they both ready for this PR?

Reply:
They'll both be excluded. They are here for debugging.